cli/diagnostics.rs

```diff
@@ -39,7 +39,6 @@ const UNSTABLE_DENO_PROPS: &[&str] = &[
   "UnixConnectOptions",
   "UnixListenOptions",
   "addSignalListener",
-  "applySourceMap",
```

Comment:
I think we need to update the declaration file too?

Reply:
Good catch, I had `cli/dts` excluded in my search box and didn't spot it.